frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -148,6 +148,8 @@ const mapDispatchToProps = {
 class SharingSidebar extends React.Component {
   state = {
     editingMode: "unknown",
+    // use this to know where to go "back" to
+    returnMode: undefined,
```

Comment:
Should we consolidate to using either `"unknown"` or `undefined`? My preference is to avoid the magic string, but I guess every other state is a magic string so :man_shrugging:

Reply:
+1 to consolidating on undefined.